final_prep/README.md

```diff
@@ -19,37 +19,42 @@ Some things to consider as you refactor include...
 
 Take your time as you go back and refactor your exercises from each section. We've included a handy checklist for you to go through below.
 
-- [ ] I have refactored my `section1` exercises to the best of my ability
-- [ ] I have refactored my `section2` exercises to the best of my ability
-- [ ] I have refactored my `section3` exercises to the best of my ability
-- [ ] I have refactored my `section4` exercises to the best of my ability
+- [x] I have refactored my `section1` exercises to the best of my ability
+- [x] I have refactored my `section2` exercises to the best of my ability
+- [x] I have refactored my `section3` exercises to the best of my ability
+- [x] I have refactored my `section4` exercises to the best of my ability
 
 ### Time Management Prep
 In Mod 0 you've learned about different techniques for managing your time at Turing. Please create a calendar for your **first 3 weeks of Mod 1**. Feel free to make your calendar fit your style, but we suggest that your calendar should include the following:
-- [ ] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
-- [ ] Study/Project work time
-- [ ] Health + Wellness
+- [x] Standard M1 class schedule (see M1 calendar [here](https://backend.turing.io/module1/)
+- [x] Study/Project work time
+- [x] Health + Wellness
 
 When you are finished, add screenshots of your calendar so we can provide feedback if needed!
 
-- `Add Week 1 Screenshot Here`
-- `Add Week 2 Screenshot Here`
-- `Add Week 3 Screenshot Here`
+<img width="1163" alt="Screen Shot 2021-06-14 at 6 05 10 PM" src="https://user-images.githubusercontent.com/83977143/121973936-91ee2500-cd3b-11eb-9472-8ecae3290ef7.png">
+
```

Comment:
I think you are planning an appropriate amount of work time, but I will encourage you to more refine your big 'study' blocks of time. 
I wasn't great at calendaring in Turing, and when I made a big block like this for myself, I found myself getting anxious about all the different stuff I needed to do, then switching between tasks way too much.
I instead recommend devoting a specific amount of time to project work, or study, or review, while leaving some time as flexible to wrap up whatever is needed.